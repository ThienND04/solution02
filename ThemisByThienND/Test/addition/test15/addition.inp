10
1840 1220 2097 627489
3075 1118 2607 282627
1501 1134 1521 479179
2986 1208 1763 197630
2432 1220 1333 699462
3024 1172 1992 920613
3046 1112 1276 927675
2068 1170 1998 764317
3244 1203 1787 904618
2106 1120 1245 182041
