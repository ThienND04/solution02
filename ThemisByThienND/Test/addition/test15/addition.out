39
4
42
3
64
0
0
0
61
5
